src/components/HyloEditor/HyloEditor.js

```diff
@@ -149,7 +149,7 @@ const HyloEditor = React.forwardRef(function HyloEditor ({
       return editorRef.current.getText()
     },
     isEmpty: () => {
-      return editorRef.current.isEmpty
+      return editorRef.current?.isEmpty
```

Comment:
wait, shouldnt editorRef.current being not existent return true? so in that case it would be editorRef.current ? editorRef.current.isEmpty : true

Reply:
With the other changes here, there is a moment in its lifecycle where `editorRef.current` is null or undefined when this.editor.current.isEmpty() is called, and so it errors.

Reply:
I get that, it just seems like for isEmpty to return the correct value (which should be true in that case) then it needs to be rewritten as I suggested

Reply:
Ahh yes sorry, I think I just saw the first half of your comment in the email and missed the second half